### PR TITLE
Add implicit column specify required guidance for use_raw_query_with_incremental

### PR DESCRIPTION
**Pull request by y-ken**
I have added guidance to avoid `java.lang.IndexOutOfBoundsException` error
fixes https://github.com/embulk/embulk-input-jdbc/issues/215

### CODE REVIEW

**Comment by y-ken** [comment]
oh, I  have noticed I need to update other plugins README under this repo, too.
Is this understanding correct?

**Comment by dmikurube** [comment]
@y-ken Ah... I guess so. `use_raw_query_with_incremental` is implemented also in other plugins.

**Review by hiroyuki-sato — APPROVED** [review]

**Comment by y-ken** [comment]
@dmikurube yes, I have updated other files, too

File: embulk-input-db2/README.md [hunk 1]
```diff
@@ -91,14 +91,17 @@ Normally, you can't write your own query for incremental loading.
...
+* `select * from ...` statement causes `java.lang.IndexOutOfBoundsException` error.
+* Using too simple prepared statement like `:a` might cause SQL parse failure.
 
 In the following example, prepared statement `:foo_id` will be replaced with value "1" which is specified in `last_record`.
 
 ```yaml
 in:
   type: jdbc
-  query:
+  query: |
```
Comment (y-ken) [inline embulk-input-db2/README.md:103]: I have fixed minor documentation typo to handle multiple lines

**Review by dmikurube — APPROVED** [review]
Looks good. Thanks!

Let us check also with @hito4t. I'll merge this early next week if no actions.

**Comment by dmikurube** [comment]
Merging... Thanks!

**Pull request merged**